[EDU-4281] refactor: rule set > edge firewall (#878)
aziontech/docs: src/content/docs/en/pages/guides/marketplace/integrations/fingerprint.mdx

## Configuring the integration

### Setting up an edge firewall

To do so, follow the steps:

1. On the upper-left corner, select **Products menu** > **Edge Firewall** in the **SECURE** section.
2. Click the **Add Rule Set** button.
3. Give an easy-to-remember name to your edge firewall.
4. Select the domains you want to protect with the function.
5. Enable the **Edge Functions** switch in the **Edge Firewall Modules** section.
6. Click the **Save** button.

> Done. Now you've instantiated the edge firewall for your function and have access to edge functions on your edge firewall.

Reviewer: ```suggestion
Done. Now you've instantiated the edge firewall for your integration and it has access to edge functions.
```
